Header header
BoundingBox[] bounding_boxes
string qem_out
int16[] valid_list